blocks-touch/b-slider/b-slider.js

```diff
@@ -127,6 +127,11 @@
             slider._currentX = slider._currentX || 0;
             // объект с текущими параметрами каждого слайда
             slider._touch = {};
+            // страница для отслеживания pointermove / pointerup
+            slider._page = $(window);
+            // pointer-хэндлеры привязаные к контексту исполнения
+            slider._bonPointerMove = slider._onPointerMove.bind(slider);
```

Comment:
`_bonPointerMove` is not speaking name. I think you can do without it, and write just `slider._onPointerMove.bind(slider)`. There's not many places where it is used.